coreclient/src/conversations/messages/persistence.rs

```diff
@@ -48,6 +48,10 @@ impl VersionedMessage {
 
     fn to_mimi_content(&self) -> anyhow::Result<MimiContent> {
         match self.version {
+            1 => {
+                let old = PhnxCodec::from_slice::<MimiContentV1>(&self.content)?;
+                Ok(old.upgrade())
```

Comment:
Maybe we can save the upgraded event permanently. In that case, we should write a migration that goes through all events in the db and upgrades them.

Reply:
This is now done in data migration when the database is opened and the message status migration is not yet applied.

Reply:
Does that mean we don't need this code anymore?